business/istio_config.go

```diff
@@ -96,7 +97,13 @@ var newSecurityConfigTypes = []string{
 
 // GetIstioConfigList returns a list of Istio routing objects, Mixer Rules, (etc.)
 // per a given Namespace.
-func (in *IstioConfigService) GetIstioConfigList(criteria IstioConfigCriteria) (models.IstioConfigList, error) {
+func (in *IstioConfigService) GetIstioConfigList(ctx context.Context, criteria IstioConfigCriteria) (models.IstioConfigList, error) {
+	var end observability.EndFunc
+	ctx, end = observability.StartSpan(ctx, "GetIstioConfigList",
+		observability.Attribute("package", "business"),
```

Comment:
I'd say that in the methods with "criteria" it would be good to have perhaps a helper method to pass the criteria.
Not blocker, I guess that can be consolidated in a future iteration, but it could be nice to check which criteria can show more latencies.

Reply:
Sure `Attribute` will work with anything that implements `Stringer` so I can add a `String` method to `IstioConfigCriteria`.